app/frontend/src/components/admin/SubmissionsTable.vue

```diff
@@ -107,13 +108,16 @@ export default {
           const data = response.data;
           const submissions = Object.keys(data).map(k => {
             let submission = data[k];
+            let inspectorName = (submission.inspectionStatuses[0].inspectorName !== null) ? submission.inspectionStatuses[0].inspectorName : ' - ';
```

Comment:
Can just be a falsey check 
if(submission.inspectionStatuses[0].inspectorName)

Reply:
oh yeah. i appreciate the reminder. thanks

Reply:
Could get undesired results with just the null check. Reminding myself what each condition evaluates to with that....

![image](https://user-images.githubusercontent.com/17445138/82473425-63d9db00-9a7e-11ea-9572-9913064ad239.png)

Reply:
huh. ok. i wasnt aware of some of that. i'm definitely saving this image.